sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/SplittableDoFnTest.java

```diff
@@ -427,22 +425,6 @@ public Void apply(Iterable<KV<String, Iterable<Integer>>> input) {
                 return null;
               }
             });
-
-    // Verify that more than 1 process() call was involved, i.e. that there was checkpointing.
-    PAssert.thatSingleton(
```

Comment:
Technically yes, but it's not necessary anymore. I verified it this way before only because there was no ProcessContinuation - but now the call self-suspends.